modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java

```diff
@@ -320,6 +319,8 @@ public void testUpdateDefaultObjectStateFlow() throws Exception {
 
 	@Test
 	public void testUpdateObjectStateTransitions() throws Exception {
+		_step4ListTypeEntry = _addListTypeEntry("step4");
+
```

Comment:
@murilostodolni can we move it to be right before the `for (ObjectState objectState : objectStates) ` [here](https://github.com/liferay-objects/liferay-portal/blob/b2bf5d3409d13641d0fa87654b71302599037054/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java#L342)? Or the order matters here?

See the [As Used](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DLocal-variable-declarations) topic for more context

Reply:
Perfect. I saved the link to read it all. Thank you @carolmariaabb

Reply:
@carolmariaabb We need to add the listTypeEntry before the get [here](We need to add the listTypeEntry before the get [here](https://github.com/liferay-objects/liferay-portal/blob/b2bf5d3409d13641d0fa87654b71302599037054/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java#L342:~:text=%22step4%22)). If we add after, [this](https://github.com/liferay-objects/liferay-portal/blob/b2bf5d3409d13641d0fa87654b71302599037054/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java#L342:~:text=_assertEquals(,.getObjectStateFlowId()))%3B) assert failed.

Reply:
No problem, @murilostodolni .. let's make sure Brian doesn't break this kkkkk